app/repositories/application-repository.js

```diff
@@ -18,6 +21,51 @@ async function getByEmail (email) {
       }]
     })
 }
+async function searchApplications (searchText, searchType, offset = 0, limit = 10) {
+  let query = {
+  }
+  let total = 0
+  let applications = []
+  if (searchText) {
+    switch (searchType) {
+      case 'sbi':
+        query.where = { 'data.organisation.sbi': searchText }
+        break
+      case 'ref':
+        query.where = { reference: searchText }
+        break
+      case 'status':
+        query.include = [
+          {
+            model: models.status,
+            attributes: ['status'],
+            where: { status: sequelize.where(sequelize.fn('LOWER', sequelize.col('status')), ' = ', searchText.toLowerCase()) }
```

Comment:
Have you considered using the `Op.eq` [operator](https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#operators). Isn't the status column value already set? If it was upper cased then you could just `searchText.toUpperCase()` and not call the sql function.